README.md

```diff
@@ -1,35 +1,93 @@
-# Readme
 
+# Cabify Message APP
 
-## Contract 
+Cabify's Bootcamp Backend project 
 
-### Express endpoint table
 
-- /messages 
-  - Method POST => 
-  - Response: 200 
-  - Error responses:
+## Deployement
 
-     - 400 "Need destination & message keys"
-     - 400 "Need destination key"
-     - 400 "Need message key"
-     - 500 "Temporal Server Error"
+Once downloaded, go to project's folder an run: .
 
-### Business Decisions
+(If you need to install Doker go to => [link](https://www.docker.com/get-started/) )
 
-In addition to testing user's data sent, I listen messsageapp error response to translate it to a more "user-friendly" language with error details, In case you have made a successful call, but the messageapp does not work properly internally. 
+```bash
+  docker compose up
+```
 
+### Docker-Compose structure
+
```

Comment:
Why do you need this in the readme ?
As it is already in the  docker-compose file

Reply:
Ok, I delete-it!